File: POO/TP2/Practica Python/Ej2/Cancion.py
class Cancion:
    __nombre = str;
    __autor: str;
    __duracion: float;
    
    @property
    def __nombre(self):
        return self.__nombre
    
    @__nombre.setter
    def __nombre(self, val):
        self.__nombre = val
    
    @property
    def __autor(self):
        return self.__autor
    
    @__autor.setter
    def __autor(self, val):
        self.__autor = val
    
    @property
    def __duracion(self):
        return self.__duracion
    
    @__duracion.setter
    def __duracion(self, val):
        self.__duracion = val

File: POO/TP2/Practica Java/EJ2/Cancion.java
package EJ2;

public class Cancion {
    private String nombre = "";
    private String autor = "";
    private Double duracion = 0.0;

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getAutor() {
        return autor;
    }

    public void setAutor(String autor) {
        this.autor = autor;
    }

    public Double getDuracion() {
        return duracion;
    }

    public void setDuracion(Double duracion) {
        this.duracion = duracion;
    }

    public Cancion(String nombre, String autor, Double duracion) {
        this.nombre = nombre;
        this.autor = autor;
        this.duracion = duracion;
    }

}
